src/site/_Site.jsx

```diff
@@ -125,53 +132,48 @@ class Site extends React.Component {
 
   renderApplicationBaseChildren() {
     const {
-      siteConfig, applicationNavigation, fetchSearchItems, basename,
+      siteConfig, applicationNavigation, fetchSearchItems,
     } = this.props;
     return (
-      <ModalManager>
-        <Switch>
-          <Route path="/raw">
-            <Raw
-              contentConfig={siteConfig.contentConfig}
-              indexPath={siteConfig.indexPath}
-            />
-          </Route>
-          <Route>
-            <DevSiteNavigation
-              siteConfig={siteConfig}
-              basename={basename}
-              onUpdateSettings={this.onUpdateSettings}
-              applicationNavigation={applicationNavigation}
-              fetchSearchItems={fetchSearchItems}
-            />
-          </Route>
-        </Switch>
-      </ModalManager>
+      <Switch>
+        <Route exact path="/" render={this.redirectSlashRoute} />
+        { siteConfig.apps.map(app => app.path && <Route path={`/${app.path}`} key={app.path} render={this.redirectToReservedRoute} />)}
+        <Route>
+          <AppSettingsContext.Provider value={this.state}>
+            <ModalManager>
+              <Switch>
+                <Route path="/raw">
```

Comment:
double checking, this allows raw routes for various application or just the default app?

Reply:
all apps, basename is excluded from subsequent routing.